fendermint/app/src/app.rs

```diff
@@ -540,6 +540,14 @@ where
             },
         };
 
+        if response.code != 0.into() {
+            tracing::error!(
```

Comment:
I wouldn't necessarily use `error` in relation to user transactions. Who knows why they failed. I worked on a project where the sys admins explained their preference for log levels:
* `error`: They get a call during the middle of the night to go fix it immediately. 
* `warn`: It's not an immediate problem but something they have to deal with before long, e.g. disk space running low. 
* `info`: For your everyday monitoring
* `debug`: Something they can enable for troubleshooting IIRC.

So I would use `info` for this, since for the operator looking at the logs, this maybe interesting but probably nothing they can do anything about. `debug` would be too verbose at the moment and hinder our own development most of the time.

Reply:
I like that categorization :) Fine by me. I was thinking about using `error` or  `warn` to make errors standout the normal operation logs. But as long as we have a common  criteria I am fine :) Fixed.